gulpfile.js

```diff
@@ -522,6 +520,7 @@ const { launch, connect } = require('hadouken-js-adapter');
 
 			let config = {
 				manifest: cfg.serverConfig,
+				onElectronClose: (exitCode)=> process.exit(exitCode),
```

Comment:
Can't we just do `onElectronClose: process.exit` here?

Reply:
Regardless of simplifying the statement, the no space before the arrow is going to trigger ESLint.

Reply:
You are correct, That should all that's needed.